src/main/java/teammates/ui/controller/InstructorCourseEditPageData.java

```diff
@@ -166,38 +137,109 @@ public int getInstructorToShowIndex() {
         return instructorToShowIndex;
     }
     
-    private ElementTag createButton(String content, String buttonClass, String id, String href,
-                                    String title, String onClick, boolean isDisabled) {
-        ElementTag button = new ElementTag(content);
-        
+    private ElementTag createAddInstructorButton(boolean isDisabled) {
+        // addInstructorButton is an actual <input> button and is thus created differently from the rest of
+        // the buttons created with <a> tags
+        ElementTag button = new ElementTag();
         button.setAttribute("type", "button");
-        
-        if (buttonClass != null) {
-            button.setAttribute("class", buttonClass);
+        button.setAttribute("id", "btnShowNewInstructorForm");
+        button.setAttribute("class", "btn btn-primary");
+        button.setAttribute("onclick", "showNewInstructorForm()");
+        if (isDisabled) {
+            button.setAttribute("disabled", null);
         }
-        
-        if (id != null) {
-            button.setAttribute("id", id);
+
+        return button;
+    }
+
+    private ElementTag createEditCourseButton(boolean isDisabled) {
+        String buttonContent = "<span class=\"glyphicon glyphicon-pencil\"></span> Edit";
+        String buttonId = "courseEditLink";
+        return createBasicButton(buttonContent, buttonId, "javascript:;", Const.Tooltips.COURSE_INFO_EDIT,
+                                 isDisabled);
+    }
+
+    private ElementTag createDeleteCourseButton(boolean isDisabled) {
+        String buttonContent = "<span class=\"glyphicon glyphicon-trash\"></span>Delete";
```

Comment:
Although it may not have difference in the UI, let's add space between the closing span and "Delete".